Anpassung sealoftrust plugin an API v2

SniperSister opened the pull request:
Änderungen sind als hotfix schon in der live deployed:
https://siwecos.de/scanned-by-siwecos?data-siwecos=www.siwecos.de

Das hier ist die Code-Änderung dazu

SniperSister committed:
Adjust seal of trust plugin to v2 api

SniperSister committed:
cs fix

SniperSister committed:
cs fixes

Lednerb commented:
Bitte Anpassung des Endpoints entsprechend https://siwecos.github.io/siwecos-business-layer/#retrieving-the-sealproof-data vornehmen

Lednerb reviewed (changes requested):
Bitte Anpassung des Endpoints entsprechend https://siwecos.github.io/siwecos-business-layer/#retrieving-the-sealproof-data vornehmen

SniperSister committed:
adjust plugin to new endpoint

https://siwecos.github.io/siwecos-business-layer/#retrieving-the-sealproof-data

SniperSister commented:
@Lednerb done

Lednerb commented on src/plugins/content/sealoftrust/sealoftrust.php:
```diff
@@ -139,7 +139,7 @@ function ($matches) use ($result, $dateFormat, $userDomain)
 				{
 					$class = preg_replace(
-						"/%S/", floor($this->getScore($result) / 10), preg_replace("/%s/", floor($result->{'Score'}), $class)
+						"/%S/", floor($result->score / 10), preg_replace("/%s/", floor($result->{'Score'}), $class)
```
Berechnungen nicht mehr notwendig, da score direkt als Wert 0-100 geliefert wird.

SniperSister replied:
An der stelle leiten wir ein class-attribut aus dem score ab, da kann ich nicht 1:1 den Score verwenden

Lednerb commented on src/plugins/content/sealoftrust/sealoftrust.php:
```diff
@@ -115,7 +115,7 @@ function ($matches) use ($result, $dateFormat, $userDomain)
 								);
 
-								return sprintf("%." . $precision . "f", round($this->getScore($result), $precision));
+								return sprintf("%." . $precision . "f", round($result->score, $precision));
```
Berechnungen nicht mehr notwendig, da score direkt als Wert 0-100 geliefert wird.

SniperSister committed:
remove precision calculation
